### PR TITLE
Add Eliom 6.3.0

### PR DESCRIPTION
**Pull request by vasilisp**
Compatible with OCaml 4.6.0.

### CODE REVIEW

**Comment by hcarty** [comment]
Thanks!

File: packages/eliom/eliom.6.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+depends: [
+  "ocamlfind"
+  "deriving" {>= "0.6"}
+  "ppx_deriving"
+  "ppx_tools" {>= "0.99.3"}
+  "js_of_ocaml" {>= "3.0"}
+  "js_of_ocaml-lwt"
+  "js_of_ocaml-ocamlbuild" {build}
+  "js_of_ocaml-ppx"
+  ("js_of_ocaml-ppx" {<= "3.0.2"} | "js_of_ocaml-ppx_deriving_json")
```
Comment (jpdeplaix) [inline packages/eliom/eliom.6.3.0/opam]: Is this line correct regarding the line above ?

Comment (vasilisp) [reply packages/eliom/eliom.6.3.0/opam]: You need `js_of_ocaml-ppx` in any case (line 18). If you have `<= 3.0.2` `js_of_ocaml-ppx_deriving_json` was included in the `-ppx` package, so you don't need anything extra. In `3.1.0` you need to install the split-off package, hence the conditional.

Comment (jpdeplaix) [reply packages/eliom/eliom.6.3.0/opam]: I see. Thanks !

**Pull request merged**